.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1  # Use the ref you want to point at
+    hooks:
+    - id: trailing-whitespace
+    - id: check-yaml
+
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.17.0
+    hooks:
+      - id: yamllint
+        args: [-c=.yamllint, -s]
+        exclude: >
+          (?x)^(
+            .github/.* |
```

Comment:
just out of curiosity: what is the reason to exclude these?

Reply:
Thank! @ltalirz 
I exclude this since I turn on the 'strict mode' which will check and make sure the yaml file start with `---`. yamllint will also check the github action setting, and ask to add it. I don't think that is needed.